stubs/resources/views/livewire/resource-select.blade.php

```diff
@@ -0,0 +1,70 @@
+<div class="flow">
+    <div x-data="{message: @entangle('message')}" aria-live="assertive">
+        <p x-text="message"></p>
+    </div>
```

Comment:
We can do this without needing to use Alpine at all:

```suggestion
    <div aria-live="assertive">
        @if($message)
        <p class="visually-hidden">
            {{ $message }}
        </p>
        @endif
    </div>
```

When the message is set to an empty string, Livewire will remove the visually-hidden paragraph. When the message is populated, Livewire will add it the div. Tested in VoiceOver and it works as expected!